packages/react-examples/.storybook/main.js

```diff
@@ -48,4 +41,19 @@ const config = {
   },
 };
 
+function getStories() {
+  const packageName = path.basename(__dirname);
+
+  if (!fs.existsSync(path.resolve(__dirname, '../src', packageName))) {
```

Comment:
This logic is ported from the https://github.com/microsoft/fluentui/pull/32074/files#diff-ab3c9c8aaeafc69909155f590e3326d509375e36844d88e2b29c8e56f78d4a3eL12